bundles/org.openhab.binding.somfytahoma/src/main/resources/OH-INF/i18n/somfytahoma.properties

```diff
@@ -224,3 +247,12 @@ channel-type.somfytahoma.target_temperature.label = Target Temperature
 channel-type.somfytahoma.target_temperature.description = The target temperature of the heating system
 channel-type.somfytahoma.temperature.label = Temperature
 channel-type.somfytahoma.temperature.description = The temperature value of the sensor
+channel-type.somfytahoma.water_heater_mode.label = Mode
+channel-type.somfytahoma.water_heater_mode.description = Describes the water heater mode
+channel-type.somfytahoma.water_heater_mode.state.option.manualEcoActive = Manual - Eco
+channel-type.somfytahoma.water_heater_mode.state.option.manualEcoInactive = Manual
+channel-type.somfytahoma.water_heater_mode.state.option.autoMode = Auto
+
+# channel types
+
+channel-type.somfytahoma.derogation_heating_mode.command.option.frost\ protection = Frost protection
```

Comment:
This is the correct line that was updated by the tool while not expected. To be moved at its correct place.